File: class_7/class_7/app/src/main/java/com/alejouribesanchez/class_7/Main14Activity.java
package com.alejouribesanchez.class_7;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class Main14Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main14);

        View layout = (View)findViewById(R.id.activity_main14);
        layout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                //startActivity(new Intent(Main14Activity.this, Main15Activity.class));
            }
        });
    }
}

File: class_7/class_7/app/src/main/java/com/alejouribesanchez/class_7/Main14Activity.kt
package com.alejouribesanchez.class_7

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class Main14Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main14)

        findViewById(R.id.activity_main14).setOnClickListener {
            //startActivity(new Intent(Main14Activity.this, Main15Activity.class));
        }
    }
}
